### PR TITLE
Make it clearer what internal TimersDescriptionAttribute ctor is for

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

**Comment by stephentoub** [comment]
This doesn't look like a valid change to me.  The ctor they were using called SR.GetResourceString. The new one doesn't. Am I missing something that makes it ok?

**Comment by marek-safar** [comment]
> Am I missing something that makes it ok?

No, I missed that detail. Pushed different change to make it more obvious.

File: src/libraries/System.ComponentModel.TypeConverter/src/System/Timers/TimersDescriptionAttribute.cs [hunk 1]
```diff
@@ -23,7 +23,11 @@ public class TimersDescriptionAttribute : DescriptionAttribute
         /// <summary>
         /// Constructs a new localized sys description.
         /// </summary>
-        internal TimersDescriptionAttribute(string description, string defaultValue) : base(SR.GetResourceString(description, defaultValue)) { }
+        internal TimersDescriptionAttribute(string description, bool unused) : base(SR.GetResourceString(description))
```
Comment (stephentoub) [inline src/libraries/System.ComponentModel.TypeConverter/src/System/Timers/TimersDescriptionAttribute.cs:26]: Did you mean to change the string to bool?

File: src/libraries/System.ComponentModel.TypeConverter/src/System/Timers/TimersDescriptionAttribute.cs [hunk 2]
```diff
@@ -23,7 +23,11 @@ public class TimersDescriptionAttribute : DescriptionAttribute
         /// <summary>
         /// Constructs a new localized sys description.
         /// </summary>
-        internal TimersDescriptionAttribute(string description, string defaultValue) : base(SR.GetResourceString(description, defaultValue)) { }
+        internal TimersDescriptionAttribute(string description, string unused) : base(SR.GetResourceString(description))
+        {
+            // Needed for overload resolution
+            System.Diagnostics.Debug.Assert(unused == null);
```
Comment (stephentoub) [inline src/libraries/System.ComponentModel.TypeConverter/src/System/Timers/TimersDescriptionAttribute.cs:29]: Nit: we generally would have a `using System.Diagnostics;` and then just `Debug.Assert(unused == null)` here.

**Review by stephentoub — APPROVED** [review]

**Pull request merged**